sstables/sstables_manager.hh

```diff
@@ -80,10 +92,17 @@ public:
     void set_format(sstable_version_types format) { _format = format; }
     sstables::sstable::version_types get_highest_supported_format() const { return _format; }
 
+    future<> close();
+private:
+    void add(sstable* sst);
+    void inactivate(sstable* sst);
```

Comment:
This function inactivates an sstable *and* destroys it -- did I understand the code correctly? Perhaps `remove()` should assert that the sstable is not active, because if it is, we should always use `inactivate()`?

Reply:
inactivate moves the sstable from the in-use phase (_active) to the inactive, undergoing destruction phase.

I'll add comments explaining it.